### ISSUE TITLE
"Martinize" -> "participle Martinizing" and "participle Martinized"

### ISSUE THREAD

**Issue by Ian-07** [tatuylonen/wiktextract#938]
For [Martinize](https://kaikki.org/dictionary/All%20languages%20combined/meaning/M/Ma/Martinize.html), wiktextract is including "participle" as part of the form text instead of the tags. Unfortunately, in this case I have no idea why it's doing this - the [Wiktionary entry](https://en.wiktionary.org/wiki/Martinize) contains no abnormalities as far as I can tell.

```
  "forms": [
    {
      "form": "Martinizes",
      "tags": [
        "present",
        "singular",
        "third-person"
      ]
    },
    {
      "form": "participle Martinizing",
      "tags": [
        "present"
      ]
    },
    {
      "form": "participle Martinized",
      "tags": [
        "past"
      ]
    }
```

**Comment by kristian-clausal** [comment]
Oh nooo... This is going to be a headache. It should!

**Comment by kristian-clausal** [comment]
Should be fixed by #942
